jwst/outlier_detection/outlier_detection_step.py

```diff
@@ -156,7 +156,6 @@ def process(self, input_data):
                         model.meta.cal_step.outlier_detection = "SKIPPED"
                 else:
                     self.input_models.meta.cal_step.outlier_detection = "SKIPPED"
-                self.skip = True
```

Comment:
Needed to remove this, because outlier_detection gets run multiple times, once for each set of source/slit data in the observation. If one set has the step abort, due to their only being 1 set of input data, we don't want the step to be skipped for all remaining sets of inputs.

Reply:
I thought I saw that once, reducing MOS data, and never got around to tracking it down.  Thanks for the fix!